src/share.js

```diff
@@ -28,6 +28,25 @@ function displayLoading(config, callback) {
 }
 
 function listen(config, callback) {
+  if (config.location.hash.match(PRISMIC_SESSION_REG)) return legacySetup(config, callback)
+
+  // from corsLink
+  function setRef(msg) {
+    if (msg.data.type !== 'previewRef') return
```

Comment:
About the coding style, we need to keep the same for the whole project. I would prefer keep using `;`.

Reply:
I'll convert to semicolons.